app/static/js/DataTablesCustom.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function() {
+  $('#tblViewHosts').DataTable({
+    "pageLength": 10,
```

Comment:
This previously referenced config.DEFAULT_DATATABLES_ENTRY_COUNT, but that's not going to work here. I just set it to 10 (the default value) for the time being. 

I can work on allowing this to be configurable in a future update.